java-logging-appinsights/README.md

```diff
@@ -70,10 +70,9 @@ TelemetryClient can be `autowired`  to implement custom telemetry metrics.
 
 #### Provide Instrumentation Key
      
-Add below to your application properties to be compatible with current shared CNP configurations.
-
+Configure below spring property with App Insights Instrumentation key. While using CNP webapp module, this is taken care. 
```

Comment:
```suggestion
Set environment variable: AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY, if you are deploying using the CNP pipeline this will be automatically added for you
```